test/AddCombinedRowToTableTest.cpp

```diff
@@ -12,24 +12,33 @@ static constexpr auto U = Id::makeUndefined();
 
 template <typename F>
 void testWithAllBuffersizes(const F& testFunction) {
-  for (auto bufferSize : ql::views::iota(0, 10)) {
-    testFunction(bufferSize);
-  }
-  testFunction(100'000);
+  auto eval = [&](int bufferSize) {
+    if constexpr (std::is_invocable_v<F, int, bool>) {
```

Comment:
This is oddly specific. Wouldn't it be nicer to write a function wrapper that converts the function accepting two arguments into a function that only accepts one argument, but calls both variants? This way the helper function stays true to its buffer size functionality.